guides/snp.md

```diff
@@ -0,0 +1,108 @@
+# SNP Guide
+
+## Platform Setup
+
+In order to launch SEV or SNP memory encrypted guests, the host must be prepared with a compatible kernel. Required changes in external components are still under development but are planned to be upstream soon.
+
+The easiest way to initialize SEV or SNP on a platform is to follow the sev-utils [guide](https://github.com/amd/sev-utils/blob/coco-202402240000/docs/snp.md). The provided script builds and installs the required host kernel version.
+
+Alternatively, refer to the [AMDESE guide](https://github.com/confidential-containers/amdese-amdsev/tree/amd-snp-202402240000?tab=readme-ov-file#prepare-host) to manually build the host kernel and other components.
+
+## Getting Started
+
+This guide covers platform-specific setup for SNP and walks through the complete flows for the different CoCo use cases:
+
+- [Container Launch with Memory Encryption](#container-launch-with-memory-encryption)
+- [Pre-Attestation Utilizing Signed and Encrypted Images](#pre-attestation-utilizing-signed-and-encrypted-images)

```

Comment:
Pre-attestation? Seems like this is leftover.

Reply:
Actually, this was a reference to a section that will be created in the next pr. I'll take this out for now, and add it back later